recipes/uppasd/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "uppasd" %}
+{% set version = "v6.0.2" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
Please do not call the package as the single output, this is not supported in conda-build, see https://github.com/conda/conda-build/issues/2438 .